Cargo.toml

```diff
@@ -27,15 +27,15 @@ all-features = true
 circle-ci = { repository = "embedded-graphics/embedded-graphics", branch = "master" }
 
 [dependencies]
-az = "1.1"
-fixed = { version = "0.5.7", optional = true, default-features = false }
-float-cmp = "0.8.0"
-micromath = { version = "1.1.0", default-features = false }
+az = "1.2.0"
+fixed = { version = "1.14.0", optional = true, default-features = false }
```

Comment:
To note, `fixed` v2 (currently alpha 😞) will make a big pile of its functions `const` according to the [changelog](https://gitlab.com/tspiteri/fixed/-/blob/master/RELEASES.md).

Reply:
That sounds great. Should we open an issue to remind ourself to update to version 2 if it is released?

Reply:
Opened #665 to track.